### ISSUE TITLE
interested in training data for controller in HYDRA

### ISSUE THREAD

**Issue by hu-my** [ControlNet/HYDRA#6]
Thank you for this interesting work! I’m particularly interested in the training data used for the controller in HYDRA. Could you tell me how much training data was used in the visual grounding and VQA tasks for HYDRA? Additionally, were these data labeled or unlabeled?

**Comment by ControlNet** [comment]
> Could you tell me how much training data was used in the visual grounding and VQA tasks for HYDRA?

The DQN training is separated for each dataset. We wait until it's converged, normally it's around thousands iterations.

> Additionally, were these data labeled or unlabeled?

It's labeled. We need to calculate the reward based on the metrics calculated from the prediction and ground truth.

**Comment by hu-my** [comment]
@ControlNet Thank you for your response!
